src/main/java/Duke.java

```diff
@@ -1,10 +1,177 @@
+import java.util.ArrayList;
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+    private static final Scanner in = new Scanner(System.in);
+    private static final ArrayList<Task> taskStore = new ArrayList<Task>();
+
+    private static final String EMPTY_TASKLIST = "404 Not Found";
```

Comment:
I like how every comment by Bim is stored as a constant at the start of the class.